encryption_works.md

```diff
@@ -270,6 +270,13 @@ When I opened this email I was prompted to type in my encryption passphrase to d
 
 ![](https://raw.github.com/micahflee/encryption-works/master/images/decrypt.png)
 
+## Encrypting Files with PGP
+
+As well as powering e-mail encryption with a desktop client like Apple Mail or Thunderbird, you can also use PGP to encrypt files on your hard-drive with your public key. This serves as a second layer of protection on top of hard-drive encryption and also allows you to use traditionally insecure cloud storage systems with a new degree of trust: with PGP-encrypted files, you can store documents on Dropbox that not even they can access or read.
```

Comment:
I still feel wary about recommending this because the default UI on many systems (for example, with GPGTools on a Mac) writes decrypted files to the same directory as the encrypted file. If you use Dropbox, for example, this makes it really easy to leak plaintext to Dropbox, which is the goal described in the last sentence.

I see are a couple of options here:

1. Keep the recommendation as-is, and add an explanation about moving files out of Dropbox before decrypting them.
2. Keep the recommendation, but remove reference to cloud storage providers like Dropbox because of this problematic UI interaction. Encrypting files with PGP is still useful in other contexts (for example, transferring on a flash drive or uploading to a "free storage" website).
3. Remove this recommendation entirely.